mungegithub/mungers/submit-queue.go

```diff
@@ -51,11 +51,11 @@ const (
 	claYesLabel            = "cla: yes"
 	claHumanLabel          = "cla: human-approved"
 
-	jenkinsE2EContext  = "Jenkins GCE e2e"
-	jenkinsUnitContext = "Jenkins unit/integration"
-	jenkinsNodeContext = "Jenkins GCE Node e2e"
-	travisContext      = "continuous-integration/travis-ci/pr"
-	sqContext          = "Submit Queue"
+	jenkinsE2EContext    = "Jenkins GCE e2e"
+	jenkinsUnitContext   = "Jenkins unit/integration"
+	jenkinsVerifyContext = "Jenkins verification"
```

Comment:
actually maybe it does need to be. I think many/most things in the queue have never run verification, right? so they'll just fall out of the queue and won't ever come back...